### DISCUSSION TITLE
I installed v1.20.0, I see v1.20.1 is available.  Do the WAU scripts self update?

### DISCUSSION THREAD

**Discussion by JasonCioffi** [Romanitho/Winget-AutoUpdate#614] [Q&A]
Hi,

I initially installed v1.20.0.  I see a new version of the script is available.  Will WAU self-update without user intervention?  Or do I need to enable something GPO-wise to check and apply an update?

Thanks in advance for your assistance.

**Comment by KnifMelti** [comment]
`Update WAU to Prerelease version` in GPO, but that's not for corporate situation

**Reply by JasonCioffi** [reply]
1.20.1 is a pre-release version?  OK.  

If a mainstream/public version releases will WAU update itself without user intervention or special settings enabled?

**Comment by KnifMelti** [comment]
Yes, https://github.com/Romanitho/Winget-AutoUpdate?tab=readme-ov-file#automatic-update
But I recommend to have full control by manual deployments
